logstash-core-event/spec/logstash/timestamp_spec.rb

```diff
@@ -81,4 +81,29 @@
       expect(t - 10).to eq(current)
     end
   end
+
+  context "identity methods" do
+    subject { LogStash::Timestamp.new }
+
+    it "should support utc" do
+      expect(subject.utc).to eq(subject)
```

Comment:
Is this really checking that the returned method is himself? I understand why you need this test, but find the condition interesting, might be there is another way in rspec to explain this check better.